app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.test.tsx

```diff
@@ -34,32 +38,37 @@ const Subject = ({
   analysisLoadingInSidebar?: boolean;
   analysisRequired?: boolean;
   experiment?: getExperiment_experimentBySlug;
-}) => (
-  <RouterSlugProvider mocks={[mock]} path="/my-special-slug/edit">
-    <AppLayoutSidebarLaunched
-      {...{
-        status: mockGetStatus({ status }),
-        analysisLoadingInSidebar,
-        analysisRequired,
-        analysisError: analysisError ? new Error("boop") : undefined,
-        analysis: withAnalysis
-          ? {
-              show_analysis: true,
-              daily: [],
-              weekly: {},
-              overall: mockAnalysis().overall,
-              metadata: mockAnalysis().metadata,
-              other_metrics: mockAnalysis().other_metrics,
-            }
-          : undefined,
-        experiment,
-        refetch: async () => {},
-      }}
-    >
-      <p data-testid="test-child">Hello, world!</p>
-    </AppLayoutSidebarLaunched>
-  </RouterSlugProvider>
-);
+}) => {
+  const mockedAnalysis =
```

Comment:
This could probably use a bit more refactoring but I didn't want too much time here😬